cli/parse_pdfs.py

```diff
@@ -473,7 +496,7 @@ def parse_file(
 def _pdf_num_pages(file: str):
```

Comment:
Return type here? 

What's the ignore for?

Reply:
can't remember actually, but changing as it doesn't seem needed